app/models/schema.rb

```diff
@@ -147,6 +147,13 @@
   field :solutions, !types[ProblemSolutionType], hash_key: :solutions
 end
 
+ContestSubmitMetricsType = GraphQL::ObjectType.define do
+  name 'ContestSubmitMetrics'
+  permit :logged_in
+  field :status, !types.String, hash_key: :status
```

Comment:
I'm not sure but I think that `hash_key: :status` is optional if matches field name. Same below